cmd/signer/main_test.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"crypto/rsa"
+	"math/big"
+	"testing"
+
+	"github.com/setlog/trivrost/cmd/launcher/resources"
+	"github.com/setlog/trivrost/pkg/signatures"
+)
+
+var testContent = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet."
+var testPrivateKey = `-----BEGIN PRIVATE KEY-----
```

Comment:
Needs comment how these keys relate to each other. Which private and public key belongs together?

Reply:
comments added. please re-check